### PR TITLE
DEV016 Create seeds for all models

### PR DESCRIPTION
**Pull request by YanPam**
Create seeds for all models
[card](https://trello.com/c/SKmJLZ9V/57-dev016-create-seeds-for-all-models)

### CODE REVIEW

File: app/models/musician_skill.rb [hunk 1]
```diff
@@ -7,19 +7,19 @@ class MusicianSkill < ApplicationRecord
   has_many :pending_musician_skills,
            through: :pending_musician_skill_users,
            class_name: 'MusicianSkill',
-           source: :musician_skills
+           source: :musician_skill
```
Comment (longbeardmen) [inline app/models/musician_skill.rb]: I feel like `source` attribute in `has_many` relations is redundant here

File: db/seeds.rb [hunk 2]
```diff
@@ -1 +1,39 @@
-User.create(nickname: 'eqe', first_name: 'qwe', second_name: 'quno', type: 'User', email: 'rynf', password: 'tntfm')
\ No newline at end of file
+User.create(nickname: 'Petya_Super', first_name: 'Petya', second_name: 'Ivanov', type: 'User', email: 'petya2006@mail.ru', password: '123123')
```
Comment (longbeardmen) [inline db/seeds.rb]: user = User.create(...)

File: db/seeds.rb [hunk 3]
```diff
@@ -1 +1,39 @@
...
+
+Genre.create(name: 'Metall', description: 'Some rock style')
+
+User.all.first.genres << Genre.all.first
```
Comment (longbeardmen) [inline db/seeds.rb]: user.genres << ...

File: db/seeds.rb [hunk 4]
```diff
@@ -1 +1,39 @@
...
+
+Organization.create(name: 'Moon wolfs', description: 'Hard rock organization')
+
+User.all.first.organizations << Organization.all.first
+
```
Comment (longbeardmen) [inline db/seeds.rb]: user.organizations << ...

File: db/seeds.rb [hunk 5]
```diff
@@ -1 +1,39 @@
-User.create(nickname: 'eqe', first_name: 'qwe', second_name: 'quno', type: 'User', email: 'rynf', password: 'tntfm')
\ No newline at end of file
+User.create(nickname: 'Petya_Super', first_name: 'Petya', second_name: 'Ivanov', type: 'User', email: 'petya2006@mail.ru', password: '123123')
+
+Genre.create(name: 'Metall', description: 'Some rock style')
+
+User.all.first.genres << Genre.all.first
+
+Organization.create(name: 'Moon wolfs', description: 'Hard rock organization')
```
Comment (longbeardmen) [inline db/seeds.rb]: organization = Organization.create(...)

File: db/seeds.rb [hunk 6]
```diff
@@ -1 +1,39 @@
...
+
+Place.create(name: 'Nemiga', latitude: '3535', longitude: '328', address: 'Mishkina street', description: 'Horable place', rules: 'No afro')
+
+Event.create(place_id: "#{Place.all.first.id}", name: 'New Year', description: 'Happy Event', start_time: 'Sun, 03 Mar 2019 14:17:01 +0000', end_time: 'Sun, 03 Mar 2019 14:17:01 +0000')
+
+Event.create(place_id: "#{Place.all.second.id}", name: 'Hallowin', description: 'Event with goast', start_time: 'Sun, 20 Mar 2019 14:17:01 +0000', end_time: 'Sun, 30 Mar 2019 14:17:01 +0000')
+
+Event.create(place_id: "#{Place.all.last.id}", name: 'Birthday', description: 'Event with cake', start_time: 'Sun, 10 Mar 2019 14:17:01 +0000', end_time: 'Sun, 11 Mar 2019 14:17:01 +0000')
+
+Organization.all.first.pending_events << Event.all.first
```
Comment (longbeardmen) [inline db/seeds.rb]: organization.pending_events <<

File: db/seeds.rb [hunk 7]
```diff
@@ -1 +1,39 @@
...
+
+Organization.all.first.disabled_events << Event.all.second
+
+Organization.all.first.approved_events << Event.all.last
+
+MusicianSkill.create(name: 'Bass')
```
Comment (longbeardmen) [inline db/seeds.rb]: bass_skill = MusicianSkill.create(name: 'Bass')
drum = MusicianSkill.create(name: 'Drum')

File: db/seeds.rb [hunk 8]
```diff
@@ -1 +1,39 @@
...
+
+MusicianSkill.create(name: 'Dram')
+
+MusicianSkill.create(name: 'Fleute')
+
+User.all.first.pending_musician_skills << MusicianSkill.all.first
```
Comment (longbeardmen) [inline db/seeds.rb]: user.pending_musician_skills << bass_skill
user.pending_misician_skills << drum_skill
etc

File: spec/controllers/musician_skills_controller_spec.rb [hunk 9]
```diff
@@ -0,0 +1,8 @@
+RSpec.describe MusicianSkillsController, type: :controller do
```
Comment (longbeardmen) [inline spec/controllers/musician_skills_controller_spec.rb]: remove this empty test

File: db/seeds.rb [hunk 10]
```diff
@@ -1 +1,39 @@
-User.create(nickname: 'eqe', first_name: 'qwe', second_name: 'quno', type: 'User', email: 'rynf', password: 'tntfm')
\ No newline at end of file
+user =User.create(nickname: 'Petya_Super', first_name: 'Petya', second_name: 'Ivanov', type: 'User', email: 'petya2006@mail.ru', password: '123123')
```
Comment (andreydro) [inline db/seeds.rb]: Use gem faker to generate content for seeds

File: app/serializers/musician_skills_serializer.rb [hunk 11]
```diff
@@ -0,0 +1,14 @@
+class MusicianSkillsSerializer < ActiveModel::Serializer
```
Comment (andreydro) [inline app/serializers/musician_skills_serializer.rb]: This file should not be here

File: db/seeds.rb [hunk 12]
```diff
@@ -1 +1,91 @@
-User.create(nickname: 'eqe', first_name: 'qwe', second_name: 'quno', type: 'User', email: 'rynf', password: 'tntfm')
+user =User.create(
```
Comment (andreydro) [inline db/seeds.rb]: space

File: db/seeds.rb [hunk 13]
```diff
@@ -1 +1,91 @@
...
+user.genres << Genre.first
+
+organization = Organization.create(
+  name: Faker::Music.band,
+  description: Faker::JapaneseMedia::SwordArtOnline.item
+  )
+
+user.organizations << Organization.all.first
+
+first_random_place = Place.create(
```
Comment (andreydro) [inline db/seeds.rb]: Use
```
3.times do 
 Place.create(params)
end
```

File: db/seeds.rb [hunk 14]
```diff
@@ -1 +1,91 @@
...
+
+thrid_random_event = Event.create(
+  place_id: "#{thrid_random_place.id}", 
+  name: Faker::FunnyName.name, 
+  description: Faker::JapaneseMedia::SwordArtOnline.item, 
+  start_time: Faker::Time.between(from: DateTime.now - 1, to: DateTime.now, format: :default), 
+  end_time: Faker::Time.between(from: DateTime.now - 1, to: DateTime.now, format: :default)
+  )
+
+organization.pending_events << first_random_event
```
Comment (andreydro) [inline db/seeds.rb]: Event.first

File: db/seeds.rb [hunk 15]
```diff
@@ -1 +1,91 @@
...
+
+organization.approved_events << thrid_random_event
+
+first_random_skill = MusicianSkill.create(name: Faker::Music.instrument)
+
+second_random_skill = MusicianSkill.create(name: Faker::Music.instrument)
+
+thrid_random_skill = MusicianSkill.create(name: Faker::Music.instrument)
+
+user.pending_musician_skills << first_random_skill
```
Comment (andreydro) [inline db/seeds.rb]: MusicianSkill.first

**Pull request merged**